web/src/components/confirm-form.tsx

```diff
@@ -28,26 +29,28 @@ export default function ConfirmForm(props: Props) {
 
   return (
     <form onSubmit={onSubmit}>
+      <Prompt message="Sentences not submitted, are you sure you want to leave?" when={true} />
+
       <h2>Confirm New Sentences</h2>
       <p>
         {`${submitted.length} sentences found.`}
       </p>
 
-      { invalidated.length > 0 && (
-        <p style={{color: 'red'}}>
+      {invalidated.length > 0 && (
```

Comment:
Thanks for cleaning these up, those slipped in in one of my previous changes.

Reply:
Including a editorconfig file (IIRC common-voice does, but not sentence-collector) would help slightly, but unfortunately the options seems limited to basic indentation, not language specific styles such as this).

These crept in after indenting and unindenting some lines. I think I correct most of them.